desktop/cmp/input/CodeInput.js

```diff
@@ -96,12 +110,20 @@ export class CodeInput extends HoistInput {
         showFormatButton: PT.bool,
 
         /** True (default) to display Fullscreen button at bottom-right of input. */
-        showFullscreenButton: PT.bool
+        showFullscreenButton: PT.bool,
+
+        /** True to display Search button at bottom-right of input. When clicked, it will expand
+         * to a toolbar component at the bottom-right of input. Will collapse when search input is
+         * cleared. Search is case-insensitive.
+         * */
+        enableSearch: PT.bool
```

Comment:
Two admittedly pretty OCD nits here:

+ I've been alphabetizing prop-types to help with scanning for / locating a known prop, and simply to provide a consistent standard across components. 
+ For the multi-line comments, pls add a newline after the opening `/**` so it's on its own line.